homeassistant/components/tts/services.yaml

```diff
@@ -40,6 +40,49 @@ say:
       selector:
         object:
 
+speak:
```

Comment:
I added this service as the existing `say` service uses an `entity_id` field in the service data parameters to target a media player entity. We need this parameter for targeting the tts entity.

Reply:
I've been thinking about deprecating those services completely. To do TTS, you use a "Play Media" service/action and use the media picker to select TTS.

Reply:
You know what, if we do that, we should do that in a future PR .